tests/test-case-bad.yaml

```diff
@@ -1,167 +1,148 @@
-apiVersion: workload.codeflare.dev/v1beta1
+apiVersion: workload.codeflare.dev/v1beta2
 kind: AppsWrapper
 metadata:
   labels:
     orderedinstance: cpu.small_gpu.large
   nam: unit-test-cluster
   namspace: ns
 spec:
-  priority: 9
```

Comment:
Quick Q. Is priority removed from AppWrapper completely?

Reply:
Yes.  We dropped all AppWrapper-specific priority support.   One would now just use Kueue's workload priority classs mechanisms (https://kueue.sigs.k8s.io/docs/concepts/workload_priority_class/) for AppWrappers just like you'd do with any other resource type that Kueue can manage.

Reply:
We probably should consider adding support for Kueue's workload priorities to the SDK, but it should be done fresh since it works differently and it would apply to everything (not just AppWrappers).